metadata-ingestion/src/datahub/ingestion/source_config/usage/snowflake_usage.py

```diff
@@ -1,78 +1,18 @@
 import logging
-from typing import Dict, Optional
+from typing import Optional
 
 import pydantic
 
```

Comment:
This is no longer needed, as we do not have separate snowflake usage source. These configs are already covered in SnowflakeConfig and removing these does not introduce backward incompatibility.

Reply:
nice!